github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -371,6 +371,42 @@ presubmits:
             requests:
               memory: "4Gi"
 
+  - name: pull-kubevirt-gosec-subset
```

Comment:
So I understand we are creating another job so that we can run subsets. At some point in the future this job will then become obsolete? Then we'll remove this one and make the other one required right?

Reply:
Yes, I want to keep the original lane so we can use it on PRs and see the converge to final switch.